lib/game.rb

```diff
@@ -0,0 +1,90 @@
+class Game
+  attr_accessor :board, :player_1, :player_2
+
+  WIN_COMBINATIONS = [
+    [0,1,2],
+    [3,4,5],
+    [6,7,8],
+    [0,3,6],
+    [1,4,7],
+    [2,5,8],
+    [0,4,8],
+    [2,4,6]
+  ]
+
+  @@all = {"X"=>0, "O"=>0, "Senseless Destruction"=>0}
+
+  def initialize(player_1=Players::Human.new("X"), player_2=Players::Human.new("O"), board=Board.new)
+    @board = board
+    @player_1 = player_1
+    @player_2 = player_2
+    puts "\nSweet. When it's your turn, enter a number 1 through 9 to place your token \non the corrosponding spot of the board, '1' being the top left corner and '9' being the bottom right.\n\n"
+  end
+
+  def current_player
+    if self.board.turn_count.even?
```

Comment:
Consider using a 1 line ternary here to make your code a little more concise and still readable. 

`self.board.turn_count.even? ? @player_1 : @player_2`